test/e2e/specs/customer_admin.go

```diff
@@ -218,5 +219,26 @@ var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fa
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	It("should sync AAD admin group", func() {
+		syncSecret, err := cli.Client.Admin.CoreV1.Secrets("openshift-infra").Get("aad-group-sync-config", metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		idPre := "44e69b"
+		idSuf := "3a8907"
```

Comment:
are you sure this is correct? the objectID ends in "730d7b0" when i go look at it

Reply:
@asalkeld, thanks.  Will update.